docs/complexity-antipatterns-and-refactoring-strategies.md

```
@@ -344,30 +344,30 @@ escalating into full-blown Bumpy Roads.
 1. **Increasing Cognitive Complexity Scores:** A rising Cognitive Complexity
    score for a method in static analysis tools is a direct indicator.8
 
-2. **Deeply Nested Logic:** Even a single area of deep nesting (more than 2-3
+1. **Deeply Nested Logic:** Even a single area of deep nesting (more than 2-3
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 2; Actual: 1; Style: 1/2/3

(MD029, ol-prefix)